pkg/kapp/app/recorded_app_changes.go

```diff
@@ -94,14 +94,15 @@ func (a RecordedAppChanges) Begin(meta ChangeMeta) (*ChangeImpl, error) {
 		StartedAt:   time.Now().UTC(),
 		Description: meta.Description,
 		Namespaces:  meta.Namespaces,
+		DiffChanges: meta.DiffChanges,
```

Comment:
seems like it might be more appropriate to truncate here since changes classes know are closer to knowing how much to truncate and why (in theory there could be impl that instead of saving things into configmaps could be saving things into elsewhere and that may not need truncation)

Reply:
Yep, that makes sense. Will make that change